frontend/src/components/transaction-manager/transaction-manager.tsx

```diff
@@ -17,7 +17,19 @@ const logger = createLogger('TransactionManager')
  * Stores an array of parsed transactions which get passed to a modal
  */
 export function TransactionManager() {
-  const [transactions, setTransactions] = useState<ParsedTx[]>([])
+  const [transactions, setTransactions] = useState<ParsedTx[]>([
+    {
```

Comment:
question (blocking): Whys this here?

Reply:
oh no.. i think i accidentally unstashed some changes here from another branch i had. thanks for the catch, removed it now